README_DROPBOX.md

```diff
@@ -0,0 +1,19 @@
+How to develop on this:
+
+Build process is as suggested in the original readme:
+
+`sudo npm install -g grunt-cli`
+`npm install` (this doesn't actually work... but doesn't seem like it needs to work?)
+`grunt build`
+
+The react-0.13.3.js and react-0.13.3.min.js we have in rSERVER right now are build/react-with-addons.js and build/react-with-addons.min.js respectively, on the v13-dropbox branch.
+
+To update:
+* make your changes, then run `grunt extract-errors` and `grunt build`
+* `cp build/react-with-addons.js metaserver/metaserver/static/javascript/external/react-0.13.3-dev.js`
+* `cp build/react-with-addons.min.js metaserver/metaserver/static/javascript/external/react-0.13.3-prod.min.js`
```

Comment:
I think I ended up nuking this - I figured we wanted to use the vanilla copy for react-0.12.0-prod in case we needed to fall back.  Probably should do the same for react 13?